resources/guest_configs/microvm-kernel-ci-x86_64-4.14.config

```diff
@@ -1,10 +1,10 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 4.14.174 Kernel Configuration
+# Linux/x86 4.14.336 Kernel Configuration
```

Comment:
Could you add how much the kernel size changes in the commit body?

Reply:
yup, will do!

Reply:
Done!